docs/source/documentation/adrs/adr-012-datahub-self-hosted.html.md.erb

```diff
@@ -0,0 +1,81 @@
+---
+owner_slack: "#data-platform-notifications"
+title: ADR-012 We will host DataHub internally
+last_reviewed_on: 2024-02-07
+review_in: 6 months
+---
+
+# <%= current_page.data.title %>
+
+## Status
+
+✅ Accepted
+
+## Context
+
+A data catalogue for the MOJ is a critical pillar of the Data Platform. Following a review of open source catalogues, [DataHub](https://datahubproject.io/) was selected as the preferred tool.
+
+Since [Acryl Data](https://www.acryldata.io/acryl-datahub) offer a SAAS version of DataHub, this ADR is concerned with the choice of how we host our catalogue.
```

Comment:
```suggestion
Since [Acryl Data](https://www.acryldata.io/acryl-datahub) offer a SaaS version of DataHub, this ADR is concerned with the choice of how we host our catalogue.
```